src/query/models/matcher.go

```diff
@@ -61,7 +61,7 @@ func NewMatcher(t MatchType, n, v []byte) (Matcher, error) {
 	return m, nil
 }
 
-func (m *Matcher) String() string {
+func (m Matcher) String() string {
```

Comment:
also changes Matches function ?

Reply:
Good point, I can change it for consistency. In theory everything should still work either way, except for any interface checks on that method (which is the problem `String()` hit).